Philips.CodeAnalysis.MsTestAnalyzers/StringConstants.cs

```diff
@@ -11,7 +11,9 @@ public static class StringConstants
 		public const string TestAttributeName = @"Test";
 		public const string EnsureAttributeName = @"Ensure";
 		public const string VerifyAttributeName = @"Verify";
-
+		public const string IsTrue = @"IsTrue";
```

Comment:
I would create more elaborate names, like IsTrueMethodName

Reply:
I don't like to put the intended use of the variable into variables names.  Is it IsTrueMethodName, IsTrueVariableName, IsTrueAssertMethodName? It artificially tries to dictate the usage.